### DISCUSSION TITLE
Allow adding `Conditions` to each entity in Glance card

### DISCUSSION THREAD

**Discussion by pedrolamas** [home-assistant/frontend#15702] [Dashboard]
I would like to have a Glance card that works by only showing entities that pass a specific condition.

That would allow to create a "warnings" style glance card that would only display the entity if a certain condition was met.

The card configuration could be something like this:

```diff
 type: glance
 title: Better names
 entities:
   - entity: binary_sensor.movement_backyard
     name: Movement
+    conditions:
+      - entity: binary_sensor.movement_backyard
+        state: "on"
   - entity: switch.wall_plug_switch
     tap_action:
       action: toggle
```

On the above example, the `binary_sensor.movement_backyard` entity would only show if its own state is "on".

I have looked into the Conditional card is an "all or nothing" kind of approach, but I think there is no way for us to have a card that shows entities in regards to their own state (or the state of an attribute).

**Comment by ildar170975** [comment]
As a workaround, currently this may be achieved by card-mod, you may either to show a gap or shift the next item to this place. Details may be asked in the dedicated card-mod Community thread.

**Comment by lazyboy0172** [comment]
This is over a year old but came up by google search today for me trying to find a way to conditionally show entities on my glance card (specifically, if door locked I want to see that, but if unlocked instead show door sensor so I see if door is open or closed. Eg if I see door closed, I know it’s closed but unlocked, if it’s locked I see that but no door status since it’s obviously closed to be locked)

anyway, google led me here but nowhere else really. Has anyone accomplished this in card-mod as suggested or know where I could ask for help on that? Also, is there a feature request posted somewhere so I could vote on that to try to add? With the recent focus on frontend and usability I’m hopeful something like this might actually see some development

**Reply by karwosts** [reply]
You can do conditional entities in a glance card with the entity filter card:

https://www.home-assistant.io/dashboards/entity-filter/

**Reply by Seaniau** [reply]
> You can do conditional entities in a glance card with the entity filter card:
> 
> https://www.home-assistant.io/dashboards/entity-filter/

Close, except the conditions specified in an Entity Filter card apply to all the entities in the card. Instead of individually being able to pick and choose which entities show and when, which is what I believe OP and I are looking for.
Good suggestion though, it may do the job for some.

**Reply by karwosts** [reply]
Entity filter has global conditions **and** also conditions for individual entities. 

https://www.home-assistant.io/dashboards/entity-filter/#options-for-entities

**Reply by Seaniau** [reply]
Ah sorry, you're right.
Now, I'm trying to filter an entity based on the state of an entity that is not displayed in this card, that's proving more difficult.
But an Entity Filter card would work for OP.

**Reply by cdmn1** [reply]
2026 bump for this, also looking for a way to filter/apply visibility to individual entities in Glance card
